src/browser/synthesis-ui.js

```diff
@@ -114,7 +115,7 @@
     const webViews = state.mode === 'edit-web-view' ?
       WebView.loadByIndex(state.webViews,
                           state.webViews.selected,
-                          {uri}) :
+                          LocationChanged({uri, timeStamp: Date.now()})) :
```

Comment:
You should not use `LocationChanged` (that is action that occurs when user navigates away by clicking url) but rather `Loader.Load({uri})`.